Library/Formula/webp.rb

```diff
@@ -11,6 +11,14 @@ class Webp < Formula
     sha256 "a96249caa1541d335ab594fdd0af221109be00baa94d82429048deb56ed88008" => :mountain_lion
   end
 
+  head do
+    url "https://chromium.googlesource.com/webm/libwebp", :using => :git
```

Comment:
If you tack `.git` onto the `url` you shouldn't need the `, :using => :git` bit.